src/ipc/tests.rs

```diff
@@ -145,15 +145,11 @@ fn test_from_device_files() -> Result<()> {
                 let _ = serde_json::from_str::<AppInstanceStatus>(&data)
                     .map_err(|err| SerdeError::new(data.to_string(), err))?;
             }
-            TestMessageType::Response => {
-            }
+            TestMessageType::Response => {}
             TestMessageType::Unknown(s) => {
                 println!("Unknown message type: {}", s);
                 panic!("Deserialization is not implemented!!!");
             }
-            _ => {
```

Comment:
Backward/forward compatibility? I mean what if new client will get old data and what if we use old client with new data?

Reply:
I'm 60% done with my research on switching to protocol-based communication in EVE with nkv and probably Protobuf, hang on there